spec/models/category_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'spec_helper'
+
+describe Category do
+  it { should have_many(:videos)}
+
+  it "saves itself" do
+    category = Category.new(name: "comedies")
+    category.save
+    expect(Category.first).to eq(category)
+  end
+
+  it "has many videos" do
```

Comment:
Its better to use should matchers for this. Other than that you are testing Rails default behavior which has been tested by Rails team.